Makefile

```
@@ -1,7 +1,7 @@
 NAME="github.com/odpf/compass"
 VERSION=$(shell git describe --always --tags 2>/dev/null)
 COVERFILE="/tmp/compass.coverprofile"
-PROTON_COMMIT := "9eaca223dfd5ca0297abdb593fa73e8522d8ef62"
+PROTON_COMMIT := "9d1d2af4192b0b31e95d417bbd17752825776a08"
```

```suggestion
PROTON_COMMIT := "ab6d8cf86c6c86c8eba0ed7dde79c36eb2acdbf5"
```

The commit on master - https://github.com/odpf/proton/commit/ab6d8cf86c6c86c8eba0ed7dde79c36eb2acdbf5

---

updated using latest proton commit https://github.com/odpf/proton/commit/bfab84cdbc61ffa8725bcef7e44635885488bf24